linebot/webhook.py

```diff
@@ -35,6 +35,40 @@
 from .utils import LOGGER, PY3
 
 
+def safe_compare_digest(target1, target2):
+    """safe_compare_digest method.
+
+    :param str target1: string for compare
+    :param str target2: string for compare
+    :rtype: bool
+    :return: result
+    """
+    str1 = str(target1)
+    right = str(target2)
+
+    len1 = len(str1)
+    len2 = len(right)
+    ret = 0
+
+    if len1 == len2:
+        left = str1
+        ret = 0
+
```

Comment:
In your hint. There is an answer. so I avoid to use else :)
    /* don't use else here to keep the amount of CPU instructions constant,
     * volatile forces re-evaluation */